keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/ImageHandler.kt

```diff
@@ -45,7 +45,12 @@ class ImageHandler(
           ArtifactCheckSkipped(artifact.type, artifact.name, "ActuationInProgress")
         )
       } else {
-        val latestArtifactVersion = artifact.findLatestArtifactVersion()
+        val latestArtifactVersion = try {
+          artifact.findLatestArtifactVersion()
+        } catch (e: NoKnownArtifactVersions) {
+          log.debug(e.message)
```

Comment:
Is this really a debug situation? It seems like it shouldn't happen.

Reply:
Why? When the artifact is first registered, you may not have any versions. That's what I'm seeing with new apps.